flink-runtime/src/main/java/org/apache/flink/runtime/io/network/netty/CreditBasedSequenceNumberingViewReader.java

```diff
@@ -210,4 +214,12 @@ public String toString() {
 			", isRegisteredAsAvailable=" + isRegisteredAsAvailable +
 			'}';
 	}
+
+	@Override
+	public EnqueueAction priorityEvent(AbstractEvent event, BufferConsumer eventBufferConsumer) {
+		// make sure netty consumer knows that it can immediately process next buffer
+		notifyDataAvailable();
```

Comment:
I think it is not necessary to call `notifyDataAvailable` here. The event actually has not been added into queue yet, so the netty thread can not fetch data immediately, even I am not sure it might bring mistake here. And after adding event into queue inside `PipelinedSubpartition`, the `notifyDataAvailable` would be called in normal way.